Makefile

```diff
@@ -1,11 +1,16 @@
+MARKDOWN = pandoc -f markdown+tex_math_dollars
+
 all: build/spec.html
 clean:
-	rm -rf build/
+	rm -rf build/ spec.pdf
```

Comment:
I assume not putting this in the `build/` directory is just because of `.gitignore`?

Reply:
Partly that, partly because it seemed like the most readable version of the spec should be easily accessible from the repository home page.